net/mwan3/files/usr/sbin/mwan3rtmon

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+set -o pipefail
```

Comment:
This is only for debug?

Reply:
Removed in 3543df28e. We can squash all the commits before merging.